enhancements/update/to-multi-arch.md

```diff
@@ -0,0 +1,121 @@
+tracking-link: # link to the tracking ticket (for example: Jira Feature or Epic ticket) that corresponds to this enhancement
+  - https://issues.redhat.com/browse/OTA-962
+see-also:
+  
+replaces:
+
+superseded-by:
+
+---
+
+# Duration of Migration To Multi Arch
+
+## Summary
+
+This enhancement provides a proposal to improve Cluster Version Operation's evaluation on the progress of [migrating an existing single-architecture cluster to multi-architecture](https://docs.openshift.com/container-platform/4.17/updating/updating_a_cluster/migrating-to-multi-payload.html).
+
+
+## Motivation
+
+During a cluster upgrade, the Cluster Version Operator (CVO) downloads the desired release payload and applies the manifests of OpenShift operators to the cluster.
+Each OpenShift operator reports the versions of its own and its operands via `ClusterOperator.status.versions` so that CVO can oversee the upgrade process of the operator.
+When all of their versions are the desired version, CVO considers that the upgrade of the operator completes.
+
+The command `oc adm upgrade --to-multi-arch` is used to migrate a cluster from a single-architecture to a multi-architecture cluster. It is implemented via a cluster upgrade by switching the underlying images to the ones from the multi-arch payload *at the current version*.
+Because `ClusterOperator.status.versions` contains the information only about the operator's version number at the moment and `--to-multi-arch` starts the upgrade targeting *the same version*, the upgrade of each operator finishes right away from the CVO's point of view. Without the ability to recognize the component's turning into multi-arch, CVO's evaluation of the upgrade progress is inaccurate, i.e., the operators' upgrade is still ongoing when CVO claims complete. This enhancement is to define a proposal to address the issue.
+
+
+### User Stories
+
+* As a cluster admin, I want CVO to claim that migrating a cluster to multi-architecture is complete correctly.
+
+
+### Goals
+
+This enhancement is to provide a proposal that addresses the issue about the upgrade progress during a cluster migration to multi-arch. The proposal has impact only after the migration to multi-arch starts.
+
+### Non-Goals
+
+Whether or not 'oc adm upgrade' is executed succesfully or the requested upgrade is accepted is not in the scope of this enhancement. So it is _not_ required to have test cases for this enhancement to cover (although they are valid in general):
+- upgrade from multi-arch to single-arch,
+- upgrade from multi-arch to multi-arch, or
+- upgrade a cluster in a cloud provider where multi-arch is not supported to multi-arch.
+
+The `oc adm upgrade status` might be useful to monitor the migration progress to multi-arch but the output of the command does not block this enhancement and its implementation. The `status` command is designed for cluster upgrades but migration is not an upgrade conceptually and is implemented through an upgrade.
+
+## Proposal
```

Comment:
nit: [CI linter wants more sections][1]:

```
enhancements/update/to-multi-arch.md missing "### Workflow Description"
enhancements/update/to-multi-arch.md missing "#### Hypershift / Hosted Control Planes"
enhancements/update/to-multi-arch.md missing "#### Standalone Clusters"
enhancements/update/to-multi-arch.md missing "#### Single-node Deployments or MicroShift"
enhancements/update/to-multi-arch.md missing "### Risks and Mitigations"
enhancements/update/to-multi-arch.md missing "## Graduation Criteria"
enhancements/update/to-multi-arch.md missing "### Dev Preview -> Tech Preview"
enhancements/update/to-multi-arch.md missing "### Tech Preview -> GA"
enhancements/update/to-multi-arch.md missing "### Removing a deprecated feature"
enhancements/update/to-multi-arch.md missing "## Upgrade / Downgrade Strategy"
enhancements/update/to-multi-arch.md missing "## Version Skew Strategy"
enhancements/update/to-multi-arch.md missing "## Operational Aspects of API Extensions"
enhancements/update/to-multi-arch.md missing "## Support Procedures"
```

You should copy those over in their usual place from [the template][2], even if the content of that section is something like:

> Not applicable; this enhancement just describes how multiple cluster components can use existing APIs to address a problem, there is nothing to graduate.

etc.  Although it's probably worth walking through the update/downgrade/version-skew bits, because we do expect those, and we want to show that the current plan will handle them gracefully.

[1]: https://prow.ci.openshift.org/view/gs/test-platform-results/pr-logs/pull/openshift_enhancements/1690/pull-ci-openshift-enhancements-master-markdownlint/1846318739113381888#1:build-log.txt%3A517
[2]: https://github.com/openshift/enhancements/blob/master/guidelines/enhancement_template.md

Reply:
Still failing on reviewers and approvers ... I will fix that later.